portfolio/src/main/java/com/google/sps/data/LoginStatus.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.data;
+
+public class LoginStatus {
+
+    // if login is true, the url is for logoff, else url is for login
```

Comment:
Nit: This seems a little confusing. If the the url is for logoff when `login` is true, you might want to rename the variable to something like `isLoggedIn` or `isLogoffUrl`.